projects/kit/components/progress/progress-bar/progress-bar.component.less

```
@@ -72,6 +72,39 @@
         height: 0.5rem;
     }
 
+    &[new] {
+        &[data-size='xs'],
+        &[data-size='s'] {
```

My mistake, I forgot that according to our design system `m`-size should be rounded too.

Could you add the following line, please ?
```suggestion
        &[data-size='s'],
        &[data-size='m'] {
```

---

Ok, I'll fix it this evening